.github/workflows/coverage.yml

```diff
@@ -12,20 +12,21 @@ env:
 
 jobs:
   coverage:
-    name: Coverage
```

Comment:
please add this name, mergify bot will check this CI by name

Reply:
Done